n26/core/templates/n26/gang_equip.html

```
@@ -32,149 +19,88 @@
-
-        Always drawn, because the library tab is always there — and a gang with
-        no lists of its own would otherwise have no way to reach it.
-
-        Links, not client state: a list is a place you can be, so it goes in the
-        URL — linkable, back-button-friendly, and the server renders the one you
-        are on.
-        {% endcomment %}
+        <p class="text-xs text-muted">
+            Orphan UI:
+            {% if orphan_ui == "a" %}
+                <c-n26.link href="{{ orphan_toggle_a }}" underline="always">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)